test/recipes/01-test_symbol_presence.t

```diff
@@ -124,7 +124,13 @@ foreach (sort keys %stlibname) {
                 # Return the result
                 $_
             }
-            grep(m|.* [BCDST] .*|, @$_);
+            # Drop any symbol starting with a double underscore, they
+            # are reserved for the compiler / system ABI and are none
+            # of our business
+            grep !m|^__|,
+            # Only look at external definitions
+            grep m|.* [BCDST] .*|,
+            @$_ ),
```

Comment:
Is this universally true? running nm ./libcrypto.a I see symbols like __rsaz_512_mul in ./crypto/bn/rsaz-x86_64.s.  Or are those filtered out by some other criteria?

Reply:
*We really shouldn't produce any symbol that starts with `__`, that belongs with the C compiler or the system*

The C standard says this quite clearly, if I remember correctly.